velox/docs/functions/presto/conversion.rst

```diff
@@ -1271,3 +1271,20 @@ Miscellaneous
         SELECT typeof(123); -- integer
         SELECT typeof(1.5); -- double
         SELECT typeof(array[1,2,3]); -- array(integer)
+
+From interval year month
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Casting from interval year month to integer returns the total number of months in the interval.
+The conversion is straightforward where 1 year = 12 months.
+
+Valid examples:
+
+::
+
+  SELECT cast(INTERVAL '1' YEAR as integer); -- 12
```

Comment:
I tested this sql in Presto while got the below exception. Is this cast supported in Presto?

> presto> SELECT cast(INTERVAL '1' YEAR as integer);
Query 20250426_172530_00000_y5qc4 failed: line 1:8: Cannot cast interval year to month to integer
SELECT cast(INTERVAL '1' YEAR as integer)